app/bundles/CoreBundle/Test/MauticMysqlTestCase.php

```diff
@@ -25,6 +26,18 @@ abstract class MauticMysqlTestCase extends AbstractMauticTestCase
      */
     protected $useCleanupRollback = true;
 
+    /**
+     * @param array<mixed> $data
+     */
+    public function __construct(?string $name = null, array $data = [], $dataName = '')
+    {
+        parent::__construct($name, $data, $dataName);
+
+        $this->configParams += [
+            'db_driver' => 'pdo_mysql',
```

Comment:
yes, I did it because there is both a `MauticSqliteTestCase` and a `MauticMysqlTestCase` class.
`MauticSqliteTestCase` uses another driver, so wouldn't be compatible.

The `MauticSqliteTestCase` is AFAICT not used (or sqlite support in general, see [forum](https://forum.mautic.org/t/mautic-with-sqlite/5020) and [code base](https://github.com/mautic/mautic/blob/4.x/app/bundles/InstallBundle/Configurator/Step/DoctrineStep.php#L167)), so could be removed in another PR.

Reply:
Gotcha. We have to remove `MauticSqliteTestCase` for M5. It's not used by anything and I bet it wouldn't even work if anyone would try to use it. I would even encourage you to remove it in this PR but don't want to push if you are not comfortable with it. Your current solution is fine.

Reply:
yeah IMO cleaning up sqlite may go further then tests only, so I rather do that separately

and indeed, no way Mautic will work OOTB on sqlite in it's current version :)